layouts/partials/head.html

```diff
@@ -10,7 +10,7 @@
 <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css"> -->
 {{ "<!-- Custom CSS -->" | safeHTML }}
 {{ "<!-- Font Awesome -->" | safeHTML }}
-<script defer src="https://use.fontawesome.com/releases/v6.1.1/js/all.js"></script>
+<script defer src="https://use.fontawesome.com/releases/v6.5.1/js/all.js"></script>
```

Comment:
```suggestion
<script defer src="https://use.fontawesome.com/releases/v6.1.1/js/all.js"></script>
```

Reply:
I'm actually happy for the Font Awesome version to be bumped, no reason it can't be